### DISCUSSION TITLE
VS-code theme

### DISCUSSION THREAD

**Discussion by PabloMelo11** [adonisjs/core#3471] [Help] [answered]
What VS-code theme was used for post-documentation?

**Comment by Julien-R44** [comment, marked as answer]
This is the "material-palenight" theme
https://github.com/adonisjs/docs.adonisjs.com/blob/60892fef5b0dcb426ca23c9feec3d98f7ba0381e/config/markdown.ts#L43

https://marketplace.visualstudio.com/items?itemName=whizkydee.material-palenight-theme

**Comment by PabloMelo11** [comment]
Thanks :)
